packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -0,0 +1,398 @@
+        returns (bool)
+    {
+        return _interfaceId == MULTISIG_INTERFACE_ID || super.supportsInterface(_interfaceId);
+    }
+
+    /// @notice Returns the support threshold parameter stored in the voting settings.
+    /// @return The support threshold parameter.
+    function minApprovals() public view virtual returns (uint256) {
+        return minApprovals_;
+    }
+
+    /// @notice Returns the number of approvals,
+    /// @param _proposalId The ID of the proposal.
+    /// @return The number of approvals.
+    function approvals(uint256 _proposalId) public view returns (uint256) {
+        return proposals[_proposalId].tally.approvals;
+    }
+
+    /// @notice Updates the minimal approval parameter.
+    /// @param _minApprovals The new minimal approval value.
+    function updateMinApprovals(uint256 _minApprovals)
+        external
+        auth(UPDATE_MULTISIG_SETTINGS_PERMISSION_ID)
+    {
+        _updateMinApprovals(_minApprovals);
+
+        emit MinApprovalUpdated({minApprovals: _minApprovals});
+    }
+
+    /// @notice Adds new members to the address list and updates the minimum approval parameter.
+    /// @param _members The addresses of the members to be added.
+    function addAddresses(address[] calldata _members)
+        external
+        auth(UPDATE_MULTISIG_SETTINGS_PERMISSION_ID)
+    {
+        _updateAddresslist(_members, true);
+
+        emit AddressesAdded({members: _members});
+    }
+
+    /// @notice Removes existing members from the address list. Previously, it checks if the new addresslist length at least as long as the minimum approvals parameter requires. Note that `minApprovals` is must be at least 1 so the address list cannot become empty.
+    /// @param _members The addresses of the members to be removed.
+    function removeAddresses(address[] calldata _members)
+        external
+        auth(UPDATE_MULTISIG_SETTINGS_PERMISSION_ID)
+    {
+        // Check if the new addresslist has become shorter than the current minimum number of approvals required.
+        uint256 newAddresslistLength = addresslistLength(0) - _members.length;
+        if (newAddresslistLength < minApprovals_) {
+            revert MinApprovalsOutOfBounds({limit: newAddresslistLength, actual: minApprovals_});
+        }
+
+        _updateAddresslist(_members, false);
+
+        emit AddressesRemoved({members: _members});
+    }
+
+    /// @notice Creates a new majority voting proposal.
+    /// @param _metadata The metadata of the proposal.
+    /// @param _actions The actions that will be executed after the proposal passes.
+    /// @param _approve_ If true, the sender will approve the proposal.
+    /// @param _tryExecution If `true`, execution is tried after the vote cast. The call does not revert if early execution is not possible.
+    /// @return id The ID of the proposal.
+    function createProposal(
+        bytes calldata _metadata,
+        IDAO.Action[] calldata _actions,
+        bool _approve_,
```

Comment:
Why ending with `_`?  Also, why all params with leading underscore, if they are used?

Reply:
See https://github.com/aragon/core/pull/167#discussion_r1054538423

Reply:
Renamed to `_approveProposal` in 565c6a37f19a61cb06c0ded738d881e4e91bdfdf